<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smsContentsDialog</class>
 <widget class="QDialog" name="smsContentsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>241</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>373</width>
    <height>241</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>373</width>
    <height>241</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sms contents setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QCheckBox" name="protoSmsEnabled">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="alarmSmsEnabled">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2" rowspan="2">
    <widget class="QTextEdit" name="statSmsEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="2" rowspan="2">
    <widget class="QTextEdit" name="alarmSmsEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="statSmsEnabled">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="protoSmsLabel">
     <property name="text">
      <string>Protocol SMS:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2" rowspan="2">
    <widget class="QTextEdit" name="protoSmsEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QWidget" name="buttonsWrapper" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="confirmationButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="statSmsLabel">
     <property name="text">
      <string>Status SMS:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QTextEdit" name="startSmsEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="alarmSmsLabel">
     <property name="text">
      <string>Alarm SMS:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QCheckBox" name="startSmsEnabled">
     <property name="text">
      <string/>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QLabel" name="startSmsLabel">
     <property name="text">
      <string>Start SMS:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
